openQA.spec

```diff
@@ -247,6 +247,7 @@ install -m 0644 %{_sourcedir}/openQA.changes %{buildroot}%{_datadir}/openqa/publ
 #
 mkdir %{buildroot}%{_localstatedir}/lib/openqa/pool/1
 mkdir %{buildroot}%{_localstatedir}/lib/openqa/cache
+mkdir %{buildroot}%{_localstatedir}/lib/openqa/webui/cache
```

Comment:
I don't think this works. mkdir will fail unless you also create (and package) %{_localstatedir}/lib/openqa/webui

Reply:
Fixed. I can try to build and install it on e212 to be sure the packaging works.

Reply:
do that